frontend/src/scenes/dashboard/DashboardItemHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Link } from 'lib/components/Link'
+import { PageHeader } from 'lib/components/PageHeader'
+import { ArrowLeftOutlined } from '@ant-design/icons'
+import { IconDashboard } from 'lib/components/icons'
+import { useValues } from 'kea'
+import { dashboardLogic } from './dashboardLogic'
+import './Dashboard.scss'
+import { insightLogic } from 'scenes/insights/insightLogic'
+
+interface Props {
+    dashboardId: number
+}
+
+export function DashboardItemHeader({ dashboardId }: Props): JSX.Element {
+    const { dashboard } = useValues(dashboardLogic({ id: dashboardId }))
+    const { dashboardItem } = useValues(insightLogic)
+
+    return (
+        <div className="dashboard-item-header">
+            <Link to={`/dashboard/${dashboardId}`}>
```

Comment:
You can actually use the `BackTo` component for consistency. Not sure if we should show the back link if you didn't visit the dashboard before. Thoughts?

Reply:
We could drop the "back" and have it say `<- To App Analytics dashboard` or maybe remove the link entirely since we're already referencing a link to it in `Viewing graph .... from...`, 

<img width="741" alt="Screen Shot 2021-05-05 at 11 24 18 AM" src="https://user-images.githubusercontent.com/25164963/117166606-87cc2480-ad94-11eb-8686-e4a17b591db7.png">